src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
Other classes put those function as a static members inside the class. This seems to be better as it avoids some potential confusion about what scopes (the namespace, the class) contains what.